internal/configs/version1/nginx-plus.tmpl

```diff
@@ -358,8 +358,15 @@ stream {
     include /etc/nginx/stream-conf.d/*.conf;
 }
 
-{{- if (.NginxVersion.PlusGreaterThanOrEqualTo "nginx-plus-r31") }}
+{{- if .NginxVersion.PlusGreaterThanOrEqualTo "nginx-plus-r33" }}
+mgmt {
+    usage_report endpoint=product.connect.nginx.com interval=24h;
+    uuid_file /var/lib/nginx/nginx.id;
+    license_token conf/license.jwt;
+    deployment_ctx /etc/nginx/reporting/tracking.info;
+}
+{{- else if .NginxVersion.PlusGreaterThanOrEqualTo "nginx-plus-r31" }}
 mgmt {
     usage_report interval=0s;
 }
-{{- end}}
+{{- end }}
```

Comment:
@j1m-ryan do you think it still makes sense to keep this default `mgmt` block here?
From what I remember, we added this to confirm that the existence of the `mgmt` block wouldn't cause any issues.

Reply:
No it doesn't, this was as a flag for us, if we merged before adding R33 to the build

Reply:
We added this for users of older NGINX Plus versions to build the image, the next release will remove this and only have the r33 related block there.